### ISSUE TITLE
FEAT: Configurable amount of rows returned per page. (currently seems to be 100 rows per page)

### ISSUE THREAD

**Issue by Jieiku** [beekeeper-studio/beekeeper-studio#941]
when using Beekeeper-studio with a large table, it appears to limit the number of returned rows to 100.

Would be excellent if there were a configuration that could be set for the number of returned rows per page.

**Comment by rathboma** [comment]
Yes! I'm tagging the settings issue for when we have a proper settings screen -  #197

**Comment by athompson-merlin** [comment]
I just ran into this problem, is there a way to control the `LIMIT` right now, without a settings screen?

**Comment by patsn** [comment]
> I just ran into this problem, is there a way to control the `LIMIT` right now, without a settings screen?

Any solutions?

**Comment by knumat** [comment]
This would be a really nice addition!

The easiest way I have found to adjust the `LIMIT` manually when you are looking at a table is to click the gear in the lower-right and select "Copy view to SQL."  This opens a new Query tab with a pre-written query for that table.  However, the Query tab doesn't have the edit/delete/navigate options that the table view does.

<img width="201" alt="image" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/29646487/f6f59015-477a-4aa0-9951-af5a4104a462">

**Comment by rathboma** [comment]
We'll be doing this soon! Mostly right now we're focused on these sorts of UI frustrations.

**Comment by al31c0** [comment]
Any news?

**Comment by mohammadazeemwani** [comment]
still nothing ??

**Comment by azmy60** [comment]
Hey everyone! Sorry for the very late comment. This feature is available via [configuration file](https://docs.beekeeperstudio.io/user_guide/configuration/#how-to-use-config-files). You can configure the amount of rows in the table view by adding this to the file:

```ini
[ui.tableTable]
pageSize = 200 
```

**Comment by notakamihe** [comment]
> Hey everyone! Sorry for the very late comment. This feature is available via [configuration file](https://docs.beekeeperstudio.io/user_guide/configuration/#how-to-use-config-files). You can configure the amount of rows in the table view by adding this to the file:
> 
> [ui.tableTable]
> pageSize = 200

just add a settings page like a normal developer. What is this ini bs?

**Comment by mohammadazeemwani** [comment]
> just add a settings page like a normal developer. What is this ini bs?  


@notakamihe it is not bs, it is a standard config file. Settings page within the application is just one type of graphical interface for it. 
Your text editor can also serve as a graphical interface for it (which is what normal developers do btw)  ¯\\_(ツ)_/¯

I do stand with introducing a widget on the database viewer interface to make this particular config more readily available and therefore result in good UX
